.github/workflows/notebooks.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run: |
           cd tutorials
           pip install -r requirements.txt
-          jb build . -W
+          jb build .
```

Comment:
why? This is what allows us to catch errors when running the notebooks during CI

Reply:
The noisy_scedc_tutorial reports a warning and trigger an error. Still not sure what triggered this.
```
WARNING (norm_resp): computed an
d reported sensitivities differ by more than 5 percent.
```